tests/integration/targets/odbc/aliases

```diff
@@ -0,0 +1,8 @@
+destructive
+shippable/posix/group1
+postgresql_db
+postgresql_privs
+postgresql_user
```

Comment:
```suggestion
```
I think these lines should be removed, since this target isn't providing the only tests for these modules. (As far as I understood an alias entry means that.)